client/my-sites/checkout/upsell-nudge/business-plan-upgrade-upsell/treatment.jsx

```diff
@@ -0,0 +1,225 @@
+import { Button, Gridicon } from '@automattic/components';
+import moment from 'moment';
+import { PureComponent } from 'react';
+import upsellImage from 'calypso/assets/images/checkout-upsell/upsell-rocket-2.png';
+import DocumentHead from 'calypso/components/data/document-head';
+import PageViewTracker from 'calypso/lib/analytics/page-view-tracker';
+
+import './style.scss';
+
+export class BusinessPlanUpgradeUpsellTreatment extends PureComponent {
+	constructor( props ) {
+		super( props );
+		this.state = { hours: 0, minutes: 0, seconds: 0 };
+	}
+
+	componentDidMount() {
+		const discountEndDate =
+			moment.utc( this.props.currentPlan?.subscribedDate ).unix() + 24 * 60 * 60;
```

Comment:
I think that `subscribedDate` is in ISO8601 format (eg: `2016-03-07T08:23:23+00:00`); if so, we could use JS's `Date.parse()` here to get a Unix timestamp without needing to utilize the (very heavy) `moment` library (and it doesn't seem like you're using `moment` for actual formatting anywhere).

Reply:
Good call! Thanks! I refactored using `Date.parse()` instead of `moment`.